### DISCUSSION TITLE
How to configure sanity studio preview to work live?

### DISCUSSION THREAD

**Discussion by IMY2-studio** [sanity-io/sanity#4465] [Sanity Studio Support]
Hey,

I am using the sanity starter template https://github.com/sanity-io/template-nextjs-personal-website

I'm getting this error on my live deployed studio with no console errors
<img width="913" alt="Screenshot 2023-05-07 at 3 35 44 pm" src="https://user-images.githubusercontent.com/102514959/236965726-9de968e9-7eca-4d87-88c5-438214c49f68.png">

Preview mode is working locally
<img width="894" alt="Screenshot 2023-05-07 at 3 34 46 pm" src="https://user-images.githubusercontent.com/102514959/236965811-25f745f3-dcab-4fea-b0b3-466e28815ff1.png">

Does anyone have any suggestions?

**Comment by JoeMatkin** [comment]
Hi, did you manage to fix this?
